src/plugins/embeddable/public/lib/containers/embeddable_child_panel.tsx

```diff
@@ -41,19 +51,66 @@ export class EmbeddableChildPanel extends React.Component<EmbeddableChildPanelPr
   constructor(props: EmbeddableChildPanelProps) {
     super(props);
     this.state = {
-      loading: true,
+      firstTimeLoading: true,
     };
 
     this.mounted = false;
   }
 
+  private getEventStatus(output: EmbeddableOutput): EmbeddablePhase {
+    if (output.error !== undefined) {
```

Comment:
There may be a chance the `output` data has `error: null` which would cause the function to return `error` as the status. I suggest changing the condition to this:
```suggestion
    if (output.error != null) {
```

Which ensures the status isn't set to `error` if the error is undefined or the value is `null`.

Reply:
Used `isNil` to check for both